src/app/widget_modules/security-scan/security-scan.module.ts

```diff
@@ -9,7 +9,8 @@ import { SecurityScanConfigComponent } from './security-scan-config/security-sca
 import { SecurityScanDeleteFormComponent } from './security-scan-delete-form/security-scan-delete-form.component';
 import { NgbTypeaheadModule } from '@ng-bootstrap/ng-bootstrap';
 import { SecurityScanDetailComponent } from './security-scan-detail/security-scan-detail.component';
-import { MatIconModule, MatTableModule } from '@angular/material/';
+import { MatTableModule } from "@angular/material/table";
```

Comment:
this import change needs to be added to the unit test for this file as well, else the unit tests do not pass. I am looking more into why @angular/material is no longer in node_modules after this upgrade. Same for the unit test for collectors.component.spec.ts